xrpl/models/requests/generic_request.py

```diff
@@ -33,9 +33,13 @@ def __init__(self: GenericRequest, **kwargs: Any) -> None:
             kwargs: All the arguments for the request.
         """
         # initialize all the dataclass stuff
+        method = RequestMethod.GENERIC_REQUEST
+        if "command" in kwargs:
+            method = kwargs["command"]
```

Comment:
Does this not fail typing? `command` isn't going to be a `RequestMethod`, and if `method` is stated then that's not used at all. 
I believe my original attempt to solve this problem was in `to_dict`, not in the initialization.

Reply:
Ah I see. Do you mean in `from_dict`? I believe `to_dict` needs this logic too in order to solve this bug.

Reply:
No, I mean `to_dict` - since that's what's used by `to_xrpl` to get the JSON submitted to rippled